pkg/pbq/store/wal/segment_test.go

```diff
@@ -90,39 +95,21 @@ func Test_encodeDecodeHeader(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			got, err := encodeHeader(tt.id)
+			tmp := t.TempDir()
+			stores := NewWALStores(testPipelineName, testVertexName, WithStorePath(tmp))
+			wal, err := stores.CreateStore(context.Background(), *tt.id)
+			assert.NoError(t, err)
+			newWal := wal.(*WAL)
+			got, err := newWal.encodeHeader(tt.id)
 			if !tt.wantErr(t, err, fmt.Sprintf("encodeHeader(%v)", tt.id)) {
 				return
 			}
 			result, err := decodeHeader(got)
 			assert.NoError(t, err)
 			assert.Equalf(t, tt.id, result, "encodeHeader(%v)", tt.id)
-		})
-	}
-}
-
-func Test_runeEncoding(t *testing.T) {
```

Comment:
Deleting this test case as I don't think we need to test functionality of "encoding/binary" library.